cli/src/commands/validate/validate.ts

```diff
@@ -24,19 +28,39 @@ export default async function validate(jsonSchemaInstantiationLocation: string,
 
         const validateSchema = ajv.compile(jsonSchema);
 
+        const spectralResult: SpectralResult = await runSpectralValidations(jsonSchemaInstantiation);
+        errors = spectralResult.errors;
+        validations = validations.concat(spectralResult.spectralIssues);
+
         if (!validateSchema(jsonSchemaInstantiation)) {
-            logger.error(`The instantiation does not match the JSON schema pattern. Errors: ${prettifyJson(validateSchema.errors)}'`);
-            exitCode = 1;
-        } else {
-            logger.info('The schema instantiation matches the json schema');
+            logger.debug(`JSON Schema validation raw output: ${prettifyJson(validateSchema.errors)}`);
+            errors = true;
```

Comment:
I wonder if we could simplify this a little bit by just throwing an error here with the error message on line 42. You could then remove that whole if statement from further below and let the catch block log the error and exit. You then wouldn't need the errors variable either. Thoughts?

Reply:
actually I just noticed you're using the errors variable at other places too, but I still think we can probably simplify this a little

Reply:
discussed offline and agreed these changes wouldn't actually simplify anything. I'm happy with these changes!